src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -896,6 +896,30 @@ public int Timeout
             }
         }
 
+        private bool _any;
+        /// <summary>
+        /// Wait until any of the processes stops, instead of waiting for all of them.
+        /// </summary>
+        [Parameter]
+        public SwitchParameter Any
+        {
+            get { return _any; }
+
+            set { _any = value; }
```

Comment:
Please remove fields and use auto properties for new parameters.

Reply:
Gladly!  There are a couple of other parameters elsewhere in the file that look like they could have been written as automatic properties but weren't, so I figured auto properties were discouraged for some reason.